src/css/global.css

```diff
@@ -1,19 +1,24 @@
+%btn {
+  @apply leading-none text-base text-c100 font-extrabold bg-c300 px-16;
```

Comment:
I kept the background there so I can visually see the buttons, however, you can remove the background here to easily overwrite it by components.